BlackNectar/BlackNectar/StoresMapViewController.swift

```diff
@@ -38,43 +67,48 @@ class StoresMapViewController: UIViewController, MKMapViewDelegate, CLLocationMa
         mapView.showsUserLocation = true
         
         guard let region = UserLocation.instance.currentRegion else {
-            print("could not unwrap region")
             return
         }
-
-        self.mapView.setRegion(region, animated: true)
-    }
-    
         
-    override func didReceiveMemoryWarning() {
-        super.didReceiveMemoryWarning()
-        // Dispose of any resources that can be recreated.
+        self.mapView.setRegion(region, animated: true)
     }
     
-    // populating stores as annotations in the mapView
-    
-    func populateStoreAnnotations() {
+    private func populateStoreAnnotations() {
         
         for store in storesInMapView {
             
-            if store != nil {
-                
-                let storeName = store.storeName
-                let address = store.address.allValues
-                let location = store.location
-                
-                let latitude = location["latitude"]
-                let longitude = location["longitude"]
-                
-                let annotation = MKPointAnnotation()
-                
-                annotation.coordinate.latitude = latitude as! CLLocationDegrees
-                annotation.coordinate.longitude = longitude as! CLLocationDegrees
+            let storeName = store.storeName
+            let address = store.address.allValues
+            let location = store.location
+            
+            let latitude = location["latitude"]
```

Comment:
Location still needs to be updated so that its `latitude` and `longitude` parameter are actual `Double` numerical types, instead of this generic Dictionary<String,String>